killmyjobs

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+running_only=false
+
+print_usage () {
+  cat << EOM
+Usage: $0 [-n nodelist] [-g] 
+    Kills all your jobs! By default, also kills those you have queued up too.
+    To only kill your running jobs, use the -g flag.
+
+Arguments:
+script    the script to be submitted to slurm via sbatch
```

Comment:
Missing usage of `-g` argument, `script` doesn't seem to be an argument for this script.

Reply:
Script isn't an arg with a flag, it's a positional arg. Does this make sense?

Reply:
No, because this is in `killmyjobs`, which doesn't use `script` anywhere.